java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -2378,28 +2368,22 @@ class DatabaseImpl(
     private val VERSION_6_MIGRATION = arrayOf(
       "ALTER TABLE entity_refs ADD COLUMN is_hard_ref INTEGER;"
     )
-    private val VERSION_7_MIGRATION = arrayOf(
```

Comment:
You can delete `VERSION_5_MIGRATION` as well, it's not needed. Hopefully you can just do `emptyArray()` (with a comment explaining why)

Reply:
I was erring on the side of caution, but with this encouragement will delete away :)

Thanks